sorts/stooge_sort.py

```diff
@@ -0,0 +1,37 @@
+def stoogesort(arr):
+    stooge(arr,0,len(arr)-1)
+
+def stooge(arr, i, h):
+
+    """
+
+>>> arr = [2, 4, 5, 3, 1]
+>>> stoogesort(arr)
+>>> print(arr)
+[1, 2, 3, 4, 5]
+
+"""

```

Comment:
The doctest for __stooge_sort()__ should be in the __stooge_sort()__ function, not in the __stooge()__ function.

Let's move lines 6, 8, 9, 10, 11, and 13 up to just below line 1.  The blank lines 7 and 12 can be deleted.  It should look like:
```python
def stooge_sort(arr):
    """
    >>> arr = [2, 4, 5, 3, 1]
    >>> stoogesort(arr)
    >>> print(arr)
    [1, 2, 3, 4, 5]
    """
    stooge(arr, 0, len(arr) - 1)


def stooge(arr, i, h):
```

Reply:
Done